docs/spark-queries.md

```diff
@@ -335,6 +347,18 @@ Note:
 
 2. The partitions metadata table shows partitions with data files or delete files in the current snapshot. However, delete files are not applied, and so in some cases partitions may be shown even though all their data rows are marked deleted by delete files.
 
+### Positional Delete Files
+
+To show all positional delete files from a table:
```

Comment:
only from current snapshot? If it is not referenced by current snapshot, it won't show right? I think we need to clarify